parlai/messenger/core/messenger_manager.py

```diff
@@ -618,10 +638,9 @@ def observe_message(self, receiver_id, text, quick_replies=None):
         return self.message_sender.send_fb_message(receiver_id, text, True,
                                                    quick_replies=quick_replies)
 
-    def observe_payload(self, receiver_id, data, quick_replies=None):
```

Comment:
This and the below are occurring because the internal parlai hasn't been synced since that change. Keep what it was before.